src/Typeahead.purs

```diff
@@ -344,6 +348,27 @@ asyncMulti { async, itemToObject, renderFuzzy } props =
       (defRenderContainer renderFuzzy)
   }
 
+asyncMultiInput
+  :: ∀ action item m
+  . Eq item
+  => Effect.Aff.Class.MonadAff m
+  => DefaultAsyncTypeaheadInput item m
+  -> Array (Halogen.HTML.IProp DOM.HTML.Indexed.HTMLinput (CompositeAction action Array item m))
+  -> Ocelot.Components.MultiInput.Component.Input
+  -> Input action Array item m
+asyncMultiInput { async, itemToObject, renderFuzzy } props input =
+  { items: Network.RemoteData.NotAsked
+  , insertable: NotInsertable
+  , keepOpen: false
+  , itemToObject
+  , debounceTime: Just $ Data.Time.Duration.Milliseconds 300.0
+  , async: Just async
+  , disabled: isDisabled props
+  , render: renderMultiInput
+      input
+      (defRenderContainer renderFuzzy)
```

Comment:
🎨  Can we sort these?

Reply:
And shouldn't we cast `item` to `String` here and use `Insertable identity`? Even if we're not relying on all of the `Insertable` functionality, it should still be useful to have a way of always providing `MultiInput` with a `String` value.

Reply:
> :art: Can we sort these?

Well I copied and pasted from other variants. I guess I can sort them all.

> And shouldn't we cast `item` to `String` here and use `Insertable identity`? Even if we're not relying on all of the `Insertable` functionality, it should still be useful to have a way of always providing `MultiInput` with a `String` value.

Well even though at the moment `MultiInput` can only handle `String` values, to work around current `item` parametrization in `Typeahead` we assume `itemToObject :: item -> Foreign.Object.Object String` can convert `item` to a `String` label (by taking the first item) to display in `MultiInput`. The implementation of `embeddedHandleMultiInput` and `replaceSelected` rely on this assumption. So we don't have to specialize `item` to `String` here.